codespell_lib/data/dictionary.txt

```diff
@@ -13936,7 +13996,9 @@ entaglements->entanglements
 entended->intended
 entension->extension
 entensions->extensions
+entent->intent
```

Comment:
Also `entending->intending`?

Reply:
Wait, `entend` is an obsolete form of `intend`!

In any case, either have both or remove both, or move to _rare_:
```
entended->intended
entending->intending
```

Reply:
I can't find them in any modern dictionary, so I believe putting them in `dictionary.txt` should be fine. For what it's worth, I don't recall ever having seen that spelling before either.